card/mod/01_history/set/all/actions.rb

```diff
@@ -0,0 +1,47 @@
+# -*- encoding : utf-8 -*-
```

Comment:
Right, it's about load order.

> My request is that we address the selection of modules at the same time we handle ordering. 

that's a little different.

You're talking about picking and choosing mods (analagous to a Gemfile), we're talking about dependency tracking, which needs to be represented in the mods themselves (analagous to a gemspec).

Why should we conflate those two?